theano/gof/opt.py

```diff
@@ -1131,13 +1131,20 @@ class LocalMetaOptimizer(LocalOptimizer):
 
     """
 
-    def __init__(self, tracks=None, optimizers=()):
-        self._tracks = tracks
+    def __init__(self, optimizers=()):
+        self._tracks = [x for o in optimizers for x in o.tracks()]
         self.optimizers = list(optimizers)
         self.verbose = config.metaopt.verbose
+        self.track_dict = defaultdict(lambda: [])
+        self.tag_dict = defaultdict(lambda: [])
```

Comment:
Now this code will not fill in tag_dict and track_dict for initial optimizers.

Since it appears that this is not used anywhere, can you just get rid of the `optimizers` argument in `__init__`.